bindings/pyroot_experimental/PyROOT/python/ROOT/__init__.py

```diff
@@ -18,6 +19,18 @@ def pythonization(fn):
     """
     cppyy.py.add_pythonization(fn)
 
+def instant_pythonization(fn):
+    """
+    Pythonizor decorator to be used in pythonization modules for instant pythonizations.
```

Comment:
Instead of "instant", maybe "pythonizations done upfront"? Or greedy as opposed to lazy?

Reply:
I was using the convention used in RDF. If you think "greedy" is better, we could use it.

Reply:
What about none of the two? We could have a parameter of `@pythonization` that is called `lazy`, which is by default `True` and can be set to `False` to make it instant/greedy/eager or whatever.